### PR TITLE
Merging changes from branch PiZero-WiFi into master

### PR DESCRIPTION
**Pull request by vnemes**

### CODE REVIEW

**Comment by dnutiu** [comment]
oh wow, that's a big one

File: BLECarPiZeroW/dbusObjects/GPSSensorDBUS.py [hunk 1]
```diff
@@ -32,7 +32,10 @@ def _startSensor(self):
     def getCoords(self):
         """returns the GPS object from the sensor"""
         gps = self.gpsSensor.get_gps()
-        retVal={'latitude':gps.latitude, 'longitude':gps.longitude, 'altitude':gps.altitude}
+        if gps.latitude:
+            retVal={'latitude':gps.latitude, 'longitude':gps.longitude, 'altitude':gps.altitude,'real':1.0}
+        else:
```
Comment (dnutiu) [inline BLECarPiZeroW/dbusObjects/GPSSensorDBUS.py]: I suppose this is for debugging only

**Comment by dnutiu** [comment]
feel free 2 mergw

File: BLECarPiZeroW/dbusObjects/GPSSensorDBUS.py (hunk 1 above)
Comment (vnemes) [reply BLECarPiZeroW/dbusObjects/GPSSensorDBUS.py]: If the default value is returned dBus complains about the type returned, blocking the endpoint.
Something more elegant could be done, ofc

**Pull request merged**